common/extract-examples.rb

```diff
@@ -24,6 +24,10 @@
 
 PREFIXES = {
   dc:     "http://purl.org/dc/terms/",
+  dct:    "http://purl.org/dc/terms/",
```

Comment:
I do not really understand Ruby, but shouldn't 'dc' be changed to elements, too?

Reply:
This is there for backwards compatibility. Previously, we used “dc” for dcterms. This use was pretty pervasive. Now, we only use “dcterms” and “dc11”, so this code shouldn’t come into play.

Reply:
ok!